src/main/scala/it/pagopa/interop/purposemanagement/model/persistence/serializer/v1/protobufUtils.scala

```diff
@@ -39,7 +40,9 @@ object protobufUtils {
       description = protobufPurpose.description,
       riskAnalysisForm = riskAnalysisForm,
       createdAt = createdAt,
-      updatedAt = updatedAt
+      updatedAt = updatedAt,
+      isFreeOfCharge = protobufPurpose.isFreeOfCharge.getOrElse(true),
+      freeOfChargeReason = protobufPurpose.freeOfChargeReason
```

Comment:
wait for @beetlecrunch confirmation about the default value of `freeOfChargeReason` when `isFreeOfCharge == true`

Reply:
+1

Reply:
The default value is `Sono una Pubblica Amministrazione`

Reply:
🤔

Reply:
@beetlecrunch 

For PresistentPurpose is right in this way?

```
freeOfChargeReason =
        if (isFreeOfCharge) protobufPurpose.freeOfChargeReason.orElse("Sono una Pubblica Amministrazione".some)
        else None
```

Reply:
LGTM, but may I suggest putting the default value in a final val?